lib/ExpensiMark.js

```diff
@@ -514,7 +515,7 @@ export default class ExpensiMark {
             }
 
             // We don't want to apply link rule if match[1] contains the code block inside the [] of the markdown e.g. [```example```](https://example.com)
-            if (isDoneMatching || match[1].includes('<pre>')) {
+            if (isDoneMatching || match[1].includes('<pre>') || match[1].includes('</h1>')) {
```

Comment:
Can we updete the comment above the condition to include our case - h1 as well?

Reply:
@robertKozik it's added, please check